release-pipeline.yml

```diff
@@ -0,0 +1,1956 @@
+              container: 'test-cnt-ubn-20'
+              AgentName: 'blobfuse-ubn20-arm64'
+
+        pool:
+          name: "blobfuse-ubn-arm64-pool"
+          demands:
+            - ImageOverride -equals $(agentName)
+
+        variables:
+          - name: root_dir
+            value: '$(System.DefaultWorkingDirectory)'
+          - name: work_dir
+            value: '$(System.DefaultWorkingDirectory)/azure-storage-azcopy'
+          - name: archives
+            value: $(root_dir)/archives
+
+        steps:
+          #TODO: remove this after merging to main
+          - script: |
+              wget https://raw.githubusercontent.com/dphulkar-msft/azure-storage-azcopy/az-pipelineTest/go_installer.sh -P $(work_dir)/
+              chmod 777 $(work_dir)/go_installer.sh
+              sudo ls -lRt $(work_dir)/
+            displayName: 'list'
+
+          # Custom script to install Go-lang
+          - task: ShellScript@2
+            inputs:
+              scriptPath: "$(work_dir)/go_installer.sh"
+              args: "$(root_dir)/ $(AZCOPY_GOLANG_VERSION)"
+            displayName: "Installing Go tools"
+
+          - script: |
+              echo 'Running GO Vet'
+              go vet 
+            displayName: 'Golang Vet - Linux'
+            workingDirectory: $(root_dir)
+
+          # install dependencies required for compiling azcopy
+          - script: |
+              sudo apt-get clean
+              sudo apt-get update --fix-missing
+              sudo apt-get install pkg-config libsecret-1-dev wget -y
+              sudo apt-get install ruby ruby-dev rubygems libglib2.0-dev build-essential rpm -y
+              sudo gem install dotenv -v 2.8.1
+              sudo gem install fpm -V
+            displayName: "Installing Dependencies"
+
+          # get glibc version with which build is done
+          - script: |
+             ldd --version
+            displayName: "GLIBC Version"
+
+          - script: |
+              GOARCH=arm64 GOOS=linux go build -tags "netgo" -o "$(root_dir)/azcopy_linux_arm64"
+            displayName: 'Generate Linux ARM64'
+          
+          - script: |
+              GOARCH=arm64 GOOS=linux go build -tags "netgo,se_integration" -o "$(root_dir)/azcopy_linux_se_arm64"
+            displayName: 'Generate Linux ARM64 SE Integration'
+
+          - script: |
+              sudo ls -lRt $(root_dir)/
+              cp -r $(root_dir)/azcopy_linux* $(Build.ArtifactStagingDirectory)
+            displayName: 'List the binaries and add to staging directory'
+
+          - task: PublishBuildArtifacts@1
+            inputs:
+              artifactName: 'azCopy-binaries'
+            displayName: 'Publish binaries'
+            condition: succeededOrFailed()
+          
+          - script: |
+              rm -rf $(Build.ArtifactStagingDirectory)/azcopy*
+            displayName: "Remove binaries from staging directory"
+
+          - script: |
+              export GOARCH=arm64
```

Comment:
This seems like it can be extracted into its own script right? It's called multiple times

Reply:
As it is called multiple times, created a template file around it which will be responsible for extracting azcopy version for linux arm and amd.